subsys/bluetooth/audio/csis_client.c

```diff
@@ -183,43 +185,46 @@ static int member_rank_compare_asc(const void *m1, const void *m2)
 
 static int member_rank_compare_desc(const void *m1, const void *m2)
 {
-	struct bt_csis_client_set_member *member_1 = *(struct bt_csis_client_set_member **)m1;
-	struct bt_csis_client_set_member *member_2 = *(struct bt_csis_client_set_member **)m2;
-	struct bt_csis *inst_1;
-	struct bt_csis *inst_2;
-
-	/* TODO: lookup_instance_by_set_info should be able to work on a const
-	 * member, but it relies on `bt_conn_index` which doesn't take a const
-	 * bt_conn.
+	/* If we call the "compare ascinding" function with the members
```

Comment:
ascinding -> ascending

Neat simplification, though.  Could this rather be done in the previous commit, where the functions are introduced?